curvefs/src/client/rpcclient/metacache.cpp

```diff
@@ -492,7 +493,7 @@ bool MetaCache::SelectPartition(CopysetTarget *target) {
         target->txId = newPartitions[0].txid();
     } else {
         // random select a partition
-        int index = rand() % candidate.size();  // NOLINT
+        const auto index = butil::fast_rand() % candidate.size();
```

Comment:
butil::fast_rand() doesn't init with different seed, will generate same sequence？

Reply:
`butil::fast_rand` keeps a hidden state for each thread and init state the first time when calling the function.

you can see this from the source code

https://github.com/apache/incubator-brpc/blob/c75d53a62c181db0aec300f41787c0c0786c69d0/src/butil/fast_rand.cpp#L88